### PR TITLE
ADD herd module

### PR DESCRIPTION
**Pull request by bealdav**
bois-joli

marguerite

### CODE REVIEW

**Review by hparfr — APPROVED** [review]
LGTM

File: herd/models/specie.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    _inherit = "animal.species"
+
+    _parent_name = "parent_id"
+    _parent_store = True
+    _rec_name = "name"
+    _order = "complete_name"
+
+    name = fields.Char(index=True, required=True)
+    complete_name = fields.Char(
+        string="Familly / Specie", compute="_compute_complete_name", store=True
```
Comment (max3903) [inline herd/models/specie.py]:
```suggestion
        string="Family / Specie", compute="_compute_complete_name", store=True
```

**Review by houssine78 — APPROVED** [review]

File: herd/models/specie.py (hunk 1 above)
Comment (bealdav) [reply herd/models/specie.py]: Resolved, thanks @max3903

**Comment by bealdav** [comment]
Thanks @houssine78

**Review by max3903 — APPROVED** [review]

**Comment by LESTRAT21** [comment]
label:approved

**Comment by LESTRAT21** [comment]
label:approved

**Review by LESTRAT21 — APPROVED** [review]
Tested ans approved !

**Pull request merged**